package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@arcadia-eng/utility-connect-react",
-  "version": "0.0.1-alpha.9",
+  "version": "0.0.2-alpha.1",
```

Comment:
Major version should match utility-connect main version

Reply:
Yeahhhh this has been a point of contention today -- Are we really ready to version this as version 2.0.0 when the features have not been locked down? Still feels like alpha to me

Reply:
but went ahead and bumped -- YOLO